### PR TITLE
[JGMP-CEE Q4 2022 - Q1 2023] - Reactor, Assignment

### PR DESCRIPTION
**Pull request by jrpathfinder**

### CODE REVIEW

File: src/main/java/com/epam/learn/etl/process/model/ETLModel.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.epam.learn.etl.process.model;
+
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+@Value
+@RequiredArgsConstructor
+public class ETLModel {
```
Comment (machytkaepam) [inline src/main/java/com/epam/learn/etl/process/model/ETLModel.java]: would be better to call it Sport

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+spring:
+  datasource:
+    url: jdbc:h2:mem:mydb
```
Comment (machytkaepam) [inline src/main/resources/application.yml]: are you sure with this settings ? I think you configured it as JDBC

**Review by machytkaepam — CHANGES_REQUESTED** [review]
I think you didnt configure H2 as r2dbc + I dont see any loading and parsing data from https://sports.api.decathlon.com/sports

File: src/main/resources/application.yml (hunk 2 above)
Comment (jrpathfinder) [reply src/main/resources/application.yml]: fixed.

**Pull request merged**